discovery-infra/tests/test_day2.py

```diff
@@ -0,0 +1,37 @@
+from contextlib import suppress
+
+import day2
+import pytest
+from junit_report import JunitTestSuite
+from test_infra.consts import NetworkType, OperatorStatus
+from test_infra.utils import global_variables
+from tests.base_test import BaseTest
+from tests.config import ClusterConfig, global_variables
+from types import SimpleNamespace
+
+class Day2(BaseTest):
+    @pytest.fixture
+    def new_cluster_configuration(self):
+        return self.override_cluster_configuration()
+
+    def override_cluster_configuration():
+        config = ClusterConfig()
+        config.cluster_id = global_variables.cluster_id
+        return config
+
+    @JunitTestSuite()
+    def test_deploy_nodes_cloud(self, cluster):
+        args = SimpleNamespace()
```

Comment:
we must provide arguments in some other way than mimicking cli options
can't ``execute_day2_cloud_flow`` be made to have those arguments directly?

Reply:
add a TOOD for now